controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -1087,6 +629,23 @@ private void closeConnection(Reply reply, RawClient client) {
         }
     }
 
+    private <T extends Request & WireCommand> CompletableFuture<Reply> sendRequest(RawClient connection, long requestId, T request) {
+        return connection.sendRequest(requestId, request)
+                .exceptionally(e -> {
+                    Throwable unwrap = Exceptions.unwrap(e);
+                    if (unwrap instanceof ConnectionFailedException || unwrap instanceof ConnectionClosedException) {
+                        log.warn(requestId, "Connection dropped");
+                        throw new WireCommandFailedException(request.getType(), WireCommandFailedException.Reason.ConnectionFailed);
+                    } else if (unwrap instanceof AuthenticationException) {
+                        log.warn(requestId, "Connection dropped");
```

Comment:
This message seems unrelated with the above exception.

Reply:
fixed